pkg/services/store/object/dummy/dummy_server.go

```diff
@@ -158,27 +177,46 @@ func (i dummyObjectServer) update(ctx context.Context, r *object.WriteObjectRequ
 			ETag:    createContentsHash(r.Body),
 			Body:    r.Body,
 			Version: fmt.Sprintf("%d", prevVersion+1),
-			Comment: r.Comment,
+		}
+
+		versionInfo := &ObjectVersionWithBody{
+			Body: r.Body,
+			ObjectVersionInfo: &object.ObjectVersionInfo{
+				Version:    updated.Version,
+				Modified:   updated.Modified,
+				ModifiedBy: updated.ModifiedBy,
+				Size:       updated.Size,
+				ETag:       updated.ETag,
+				Comment:    r.Comment,
+			},
+		}
+		rsp.Object = versionInfo.ObjectVersionInfo
+
+		// When saving, it must be differnet than the head version
+		if i.ETag == updated.ETag {
+			versionInfo.ObjectVersionInfo.Version = i.Version
+			rsp.Error = &object.ObjectErrorInfo{
```

Comment:
it looks more like a info-no-action-needed rather than an error 🤔

Reply:
agree -- just not sure what better thing to show.  Perhaps just leaving out the error?  an additional status?

CREATED | UPDATED | UNCHANGED?

Reply:
created/updated/unchanged seems good, though I think most clients just care that the operation succeeded, and the current object store data is X

Reply:
I added the codes... we can remove if it feels cumbersome.